source/Nevermore/RelationalTransaction.cs

```diff
@@ -461,7 +461,7 @@ IEnumerable<T> Stream<T>(IDbCommand command, DocumentMap mapping)
                         }
                         else
                         {
-                            instance = (T) Activator.CreateInstance(instanceType);
+                            instance = (T) Activator.CreateInstance(instanceType, true);
```

Comment:
@tothegills could this be considered a breaking change?

Reply:
Yes I agree, there could be private constructors that are now preferred for instance creation.

Reply:
So, how do we move forward, should this be a flag in Nevermore that by default if off?